org.zotero.Zotero.appdata.xml

```diff
@@ -11,6 +11,15 @@
       Zotero [zoh-TAIR-oh] is a free, easy-to-use tool to help you collect,
       organize, cite, and share your research sources.
     </p>
+    <p>
+      NOTE: The user home permission is provided for compatiblity purpose.
+      Because zotero 7 supports portal file chooser, home access permission can be safely removed as follows:
+      First, move folders "~/.zotero" and "~/Zotero" to the app sandbox, typically in "~/.var/app/org.zotero.Zotero/";
+      then, remove home access premission in [flatseal](https://flathub.org/apps/com.github.tchx84.Flatseal) and restart zotero.
```

Comment:
Please also add CLI instructions here: `flatpak override --user --nofilesystem=home org.zotero.Zotero`

Reply:
Because I have added the xdg-dirs back into the permission, the command is too complicated to format well in the app description.

I can add a readme file if that is desirable.